services/explorer-indexer/src/types/rpcWs/RpcWsResponse.utils.ts

```diff
@@ -0,0 +1,110 @@
+import { Logger } from '@nestjs/common';
+import { WebSocketClient } from 'nestjs-websocket';
+import util from 'util';
+import { ResponseTypes, Topics } from './RpcWsResponse.enums';
+import {
+  BaseResponseRype,
+  NewDagBlockFinalizedResponse,
+  NewDagBlockResponse,
+  NewPbftBlockHeaderResponse,
+  NewPbftBlockResponse,
+  RpcResponseData,
+} from './RpcWsResponse.interfaces';
+
+export function checkType(object: BaseResponseRype): string {
+  if (!object) return;
+  return object.subscription
+    ? object.subscription === '0x1'
```

Comment:
Why don't we use the `Subscriptions` enum here?

Reply:
Fixed in https://github.com/Taraxa-project/taraxa-platform/pull/277/commits/5377fde50d42713cd52037847e62fedfdb430f48